modules/end-user-guide/pages/mounting-volumes-secrets-and-configmaps.adoc

```diff
@@ -0,0 +1,16 @@
+:navtitle: Mounting Volumes, Secrets and ConfigMaps
```

Comment:
I am not sure a developer is looking for documentation to mount volumes, secret and configmaps but rather:
```suggestion
:navtitle: Injecting User Credentials and Configurations
```

Reply:
We need to think about this further.

Also, I suggest avoiding the phrase `Injecting User Credentials` because it confuses me as a reader due to the fact that we already have sections about OAuth and Git Credential store Secrets.

Reply:
I can see the benefits of both titles but, In my latest commit, I went with `Injecting User Credentials and Configurations`. I'm happy to discuss further but I went with this because

1. Every section under `User Guide` has a user-perspective centric title (ex. `Selecting an IDE`, `Troubleshooting Che` etc.):
![image](https://user-images.githubusercontent.com/83611742/165398395-4286206e-2cfa-4880-8b01-416fb54ef314.png)

2. I'm not sure if having `Injecting User Credentials` would cause confusion. The Git Credential store secret docs is under this new `Injecting User Credentials and Configurations` section, and Secrets are can be generally used to store user credentials and authorization-related data anyways

Reply:
> We need to think about this further and how we can bridge the gap between users' use cases and Kubernetes resources.
> 
> Also, I suggest avoiding the phrase `Injecting User Credentials` because it confuses me as a reader due to the fact that we already have sections about OAuth and Git Credential store Secrets.

Yes, making it clear is important. And "Injecting" may not be the good term. But in today documentation this is not clear yet:

- OAuth application setup: an admin task, if it's done, users git secrets creation is automated.
- Credential secrets creation: it's a user task to avoid authentication at every workspace restart.
- Git secret creation: it's an example of credential secret creation, it can be automated with OAuth application setup.